src/main/java/sdk/sample/main.java

```diff
@@ -0,0 +1,66 @@
+package sdk.sample;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.netapp.NetAppFilesManager;
+import sdk.sample.common.ProjectConfiguration;
+import sdk.sample.common.Utils;
+
+public class main
+{
+    public static void main(String[] args)
+    {
+        Utils.displayConsoleAppHeader();
+
+        try
+        {
+            run();
+            Utils.writeConsoleMessage("ANF CRR Java sample application successfully completed");
+        }
+        catch (Exception e)
+        {
+            Utils.writeErrorMessage(e.getMessage());
+        }
+
+        // Note: this should not be here in a proper environment. I leave it here for a more compact sample that does what it needs to do and exits as soon as it finishes without waiting for other threads
+        System.exit(0);
+    }
+
+    private static void run()
+    {
+        // Getting project configuration
+        ProjectConfiguration config = Utils.getConfiguration("appsettings.json");
+        if (config == null)
+            return;
+
+        // Instantiating a new ANF management client and authenticate
+        AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
```

Comment:
Code is erroring out because the subscription ID is not assigned. where do you assign it ?